pkg/gofr/datasource/pubsub/message_test.go

```diff
@@ -227,3 +227,9 @@ func TestMessage_HostName(t *testing.T) {
 
 	assert.Equal(t, "", out)
```

Comment:
```suggestion
 	assert.Empty(t, out)
```

Not related to your PR but maybe you could fix it

Reply:
Fixed it also